python/lsst/pipe/tasks/configurableActions/_configurableActionStructField.py

```diff
@@ -271,14 +283,18 @@ def __set__(self, instance: Config,
                                        "Can only assign things that are subclasses of Configurable Action")
         instance._storage[self.name] = value
 
-    def __get__(self: T, instance: Config, owner: None = None, at: Iterable[StackFrame] = None,
-                label: str = "default"
-                ) -> Union[None, T, ConfigurableActionStruct]:
+    def __get__(
+        self,
+        instance,
+        owner=None,
+        at=None,
+        label='default'
+    ) -> ConfigurableActionStruct[ActionTypeVar]:
         if instance is None or not isinstance(instance, Config):
-            return self
+            return self  # type: ignore
         else:
-            field: Optional[ConfigurableActionStruct] = instance._storage[self.name]
-            return field
+            field: Optional[ConfigurableActionStruct] = instance._storage[self.name]  # type: ignore
+            return field  # type: ignore
```

Comment:
Looks like another case where you need overrides, and that might fix your `type: ignore`s.